src/PKSim.Presentation/Presenters/Charts/IndividualSimulationComparisonPresenter.cs

```diff
@@ -138,7 +138,7 @@ protected override void ConfigureColumns()
 
          Column(BrowserColumns.Container).Visible = true;
          Column(BrowserColumns.Container).Caption = PKSimConstants.UI.Organ;
-         Column(BrowserColumns.Container).GroupIndex = 0;
+         Column(BrowserColumns.Container).GroupIndex = -1;
```

Comment:
you need to explain this change. What does -1 do? Add a comment explaining the change please (in the code)
`We use -1 because xxx`

Reply:
I would say this is OK to be 0 as well